src/components/Star.js

```diff
@@ -0,0 +1,26 @@
+import React, { useState } from 'react';
+import StarDefault from '../assets/starDefault.svg';
+import StarSeleted from '../assets/starSeleted.svg';
+
+function Star() {
+    const [starState, setStarState] = useState('default');
+
+    const handleStarSeleted = (e) => {
+        if (starState === 'default') {
+            setStarState('seleted');
+        } else {
+            setStarState('default');
+        }
+    };
```

Comment:
1. 상태가 두 개밖에 없고 토글되는 방식이라면 boolean 타입을 사용하는 것을 추천드립니다 !
훨씬 로직이 간단해져요~

```suggestion
    const [isSelected, setIsSelected] = useState(false);

    const handleStarSeleted = () => {
        setIsSelected(!isSelected);
    };
```

2. handleStarSelected 함수에서 인자인 e 는 사용하는 곳이 없으니 삭제해주세요
사용할 때만 넣어서 쓰시면 됩니당